engine/Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM python:3.9-alpine
-RUN apk add bash python3-dev build-base linux-headers pcre-dev mariadb-connector-c-dev openssl-dev libffi-dev git
-RUN pip install uwsgi
+FROM python:3.9-alpine AS prod
 
 WORKDIR /etc/app
-COPY ./requirements.txt ./
-RUN pip install regex==2021.11.2
```

Comment:
removed `regex==2021.11.2` from the `Dockerfile` and instead placed it in `requirements.txt`. `uwsgi` was already specified in `requirements.txt` so I removed it from this file.